HashVerify/Verify-Hash.psm1

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+<#
+This module contains the function to verify expected hash of the file specified in the provided path.
+#>
+
+function Verify-Hash {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$True)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $ExpectedHash,
+
+        [Parameter(Mandatory=$True)]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript({Test-Path $_ -PathType Leaf})]
+        [string]
+        $FilePath
+    )
+
+    $fileName = [System.IO.Path]::GetFileName($FilePath)
```

Comment:
do we need a cmdlet for this simple check?

Reply:
this PR is closed. have another PR